common/tools/dev-tool/'

```diff
@@ -0,0 +1,71 @@
+{
```

Comment:
is this a file named `'` ??

Reply:
Excuse me while I remove the ghosts from my PC.

Reply:
boo